docs/user/dashboard/make-dashboards-interactive.asciidoc

```diff
@@ -76,11 +76,15 @@ The *Control type* is automatically applied for the field you selected.
 
 * To expand the width of the control to fit the available space on the dashboard, select *Expand width to fit available space*.
 
-. If you are creating an *Options list*, specify the options.
+. If you are creating an *Options list*, specify the additional settings.
 
-.. To allow users to select multiple options, select *Allow multiple selections in dropdown*.
+.. To select multiple options in the dropdown, select *Allow multiple selections in dropdown*.
 
-.. To populate the entire list of options, even when the Options list takes longer to populate than expected, select *Run past timeout*.
+.. To exclude options from appearing on the dashboard, select *Allow selections to be excluded*.
```

Comment:
This setting doesn't actually make it so that results are always excluded - it simply **hides** the include/exclude toggle when it is set to `false`, whereas when it is `true` the include/exclude toggle is visible.